internal/pkg/cli/cmd/remote/workspace/delete.go

```diff
@@ -2,30 +2,63 @@ package workspace
 
 import (
 	"fmt"
+	"time"
 
+	"github.com/keboola/go-client/pkg/sandboxesapi"
+	"github.com/keboola/go-client/pkg/storageapi"
 	"github.com/spf13/cobra"
 
 	"github.com/keboola/keboola-as-code/internal/pkg/cli/dependencies"
 	"github.com/keboola/keboola-as-code/internal/pkg/cli/helpmsg"
+	deleteOp "github.com/keboola/keboola-as-code/pkg/lib/operation/project/remote/workspace/delete"
 )
 
 func DeleteCommand(p dependencies.Provider) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   `delete`,
 		Short: helpmsg.Read(`remote/workspace/delete/short`),
 		Long:  helpmsg.Read(`remote/workspace/delete/long`),
-		RunE: func(cmd *cobra.Command, args []string) error {
+		RunE: func(cmd *cobra.Command, args []string) (cmdErr error) {
+			start := time.Now()
+
+			d, err := p.DependenciesForRemoteCommand()
+			if err != nil {
+				return err
+			}
```

Comment:
@jprochazk myslim, ze toto spadne na tom, ze nie je zadany token/host ... tj. `AskHostToken` nizsie nema vyznam, dialog sa nespusti, ... to je dovod, preco to bolo prve

Reply:
kua, já si to chtěl nejdřív zkusit než to apprůvnu a potom jsem zapomněl. 🙈 Zkusím to 😅

Reply:
Ale funguje to. 🙂 
![CleanShot 2022-10-11 at 13 34 57@2x](https://user-images.githubusercontent.com/215660/195079434-6070d9e2-8eb8-4e7a-bcc2-01f54461b793.jpg)

Reply:
:man_shrugging: :slightly_smiling_face: 
![image](https://user-images.githubusercontent.com/19371734/195080197-12230987-783f-42a5-a6d1-6932c46a979f.png)

Reply:
Asi máš token v envu, mně to padá: 
```
/code ./target/keboola-cli_linux_amd64/kbc remote workspace list
Loaded env file ".env".
Error:
  - missing Storage Api token, please use "--storage-api-token" flag or ENV variable "KBC_STORAGE_API_TOKEN"
```

Reply:
Ono to tam aj pise, ze `set from ENV.`

Reply:
Ale i když na začátek dám
```
// Ask for host and token if needed
baseDeps := p.BaseDependencies()
if err := baseDeps.Dialogs().AskHostAndToken(baseDeps); err != nil {
	return err
}
```
tak se mě to nezeptá

Reply:
Aa, jasně, máš pravdu. 🤦‍♂️

Reply:
![image](https://user-images.githubusercontent.com/19371734/195081122-475c6dd5-39d7-4c27-ad43-952edf43a7c7.png)

![image](https://user-images.githubusercontent.com/19371734/195081158-32b9905e-8031-437a-b137-cc8e0afd5530.png)

Reply:
Měl jsem v envu `KBC_STORAGE_API_TOKEN`, ale prázdný 🙂.

Reply:
Nevadi. Ono by bolo pekne, ak by sa to dalo riesit automaticky pre vsetky prikazy, ale v niektorych prikazoch sa nacitava host z `.keboola/manifest.json`, takze to nie je vzdy rovnake.

Reply:
Stejná chyba byla teda i u `remote workspace list`, tak házím opravu sem